tasks/notify.py

```diff
@@ -101,7 +106,14 @@ def send_message(_, notification_type="merge", print_to_stdout=False):
         if print_to_stdout:
             print(f"Would send to {channel}:\n{str(message)}")
         else:
-            send_slack_message(channel, str(message))  # TODO: use channel variable
+            if branch == default_branch or branch.startswith('release/'):
```

Comment:
I'm not sure what `release/` is supposed to match here.

Also, this means notifications about RC pipelines or final builds, which run on tags, would also be sent to their commit author (which isn't necessarily the person that triggered that pipeline). In these cases, we want the failures to be broadcast to all channels, since the health of these builds are important to the whole Agent group.

Reply:
I changed this condition to a regex checking for a pattern like `x.y.z...`, this should do the work